nuget/script/run

```diff
@@ -7,5 +7,19 @@ echo "NuGet native updater experiment value: $nuget_experiment_value"
 if echo "$nuget_experiment_value" | grep -q 'true'; then
     pwsh "$DEPENDABOT_HOME/dependabot-updater/bin/main.ps1" $*
 else
+    operation=$1
+    dot_dependabot_directory="$DEPENDABOT_HOME/.dependabot"
+    discovery_map_path="$dot_dependabot_directory/discovery_map.json"
```

Comment:
These checks are just to see what exists on disk before and after the processes are started/ended to help narrow down when and where the files are getting deleted.